helmcharts/README.md

```diff
@@ -0,0 +1,33 @@
+# Eclipse Che Helm Charts
+
+- [Charts](#charts)
+  - [Kubernetes Versions](#kubernetes-versions)
+  - [Helm Versions](#helm-versions)
+  - [Helm Chart Install](#helm-chart-install)
+
+
+## Charts
+
+Helm charts to deploy [Eclipse Che](https://www.eclipse.org/che/)
+
+### Kubernetes Versions
+
+Minimal Kubernetes version is 1.19
+
+### Helm Versions
+
+Helm charts are only tested with Helm version 3.7.1
```

Comment:
I think we should try and detect minimal version and put something like `3.x.y+` here.